lib/krikri/registry.rb

```diff
@@ -0,0 +1,45 @@
+module Krikri
+
+  ##
+  # Provide a registry of defined items that can be looked up by a token
```

Comment:
A string is going to come from the database, when you retrieve the HarvestSource record for an institution (harvest_sources.source_type).  It just seemed more direct to expect a string for that reason.  I know that calling String.to_sym would be pretty trivial but it would be an extra step, possibly confusing to someone.